docs/dotnet-configuration.md

```diff
@@ -13,7 +98,7 @@ Additionally, it is possible to set an environment variable as metadata value fo
 ### Using Environment Variables
 
 ```sh
-export SCOPE_METADATA="sample.key1=$SAMPLE_VAR1,sample.key2=$SAMPLE_VAR2,sample.key3=sampleValue3"
+SCOPE_METADATA="sample.key1=$SAMPLE_VAR1,sample.key2=$SAMPLE_VAR2,sample.key3=sampleValue3"
```

Comment:
Please check in your agents if the `"` character is trimmed. In the Java agent, I had to fix this issue.